cves/CVE-2019-11358.yml

```diff
@@ -128,8 +155,10 @@ autodiscoverable:
 
     The answer field should be boolean. In answer_note, please explain
     why you come to that conclusion.
-  answer_note:
-  answer:
+  answer_note: |
+    A XSS test could've attempted to modify the proto object of a class and, in
```

Comment:
To me it seems a bit of a stretch that a automated tool would test for this, thoughts?